reliquery/relic.py

```diff
@@ -298,9 +298,8 @@ def add_file_from_path(self, name: str, path: str) -> None:
         self.assert_valid_id(name)
 
         with open(path, "rb") as input_file:
-
-            fileSize = getsizeof(input_file)
             buffer = BytesIO(input_file.read())
+            fileSize = buffer.getbuffer().nbytes
```

Comment:
This is an improvement, but we should make a TODO at some poitn to avoid read()ing the entire file into memory. In theory the we should be able to push binary objects that represent a stream of information. The file may be very large so reading it into memory can cause issues some times.

I'll make an issue.

Reply:
ok thanks